scripts/install/opt/seagate/cortx/motr/bin/motr_mini_prov.py

```diff
@@ -101,7 +102,7 @@ def validate_file(file):
 def is_hw_node(self):
     try:
         node_type = Conf.get(self._index,
-                    f'cluster>{self._server_id}')['node_type']
+                             f'server_node')[machine_id]['type']
```

Comment:
Please fix this Codacy warning.
If there is no placeholder in a f-string, no need to have such a f-string. Plain string is enough.

Reply:
As I see, these changes are due to the changes in the conf schema. Am I right?

The schema should be defined carefully and not change it frequently.

Also, this "f'server_node')[machine_id]" id is used in multiple places. Can we define a function to get the "node", and use it everywhere?

Reply:
Addressed review comments in e65902b..936a3f5